client/core/core.go

```diff
@@ -7001,14 +7095,34 @@ func (c *Core) authDEX(dc *dexConnection) error {
 		localBondMap[bondKey(dbBond.AssetID, dbBond.CoinID)] = struct{}{}
 	}
 
+	var unknownBondStrength uint32
+	unknownBondAssetID := -1
 	for _, bond := range result.ActiveBonds {
 		key := bondKey(bond.AssetID, bond.CoinID)
 		if _, found := localBondMap[key]; found {
 			continue
 		}
-		symb := dex.BipIDSymbol(bond.AssetID)
-		bondIDStr := coinIDString(bond.AssetID, bond.CoinID)
-		c.log.Warnf("Unknown bond reported by server: %v (%s)", bondIDStr, symb)
+		// Server reported a bond we do not know about.
+		ubs, ubaID := c.findBond(dc, bond)
+		unknownBondStrength += ubs
+		if unknownBondAssetID != -1 && uint32(unknownBondAssetID) != ubaID {
+			c.log.Warnf("Found unknown bonds for different assets. %s and %s.",
```

Comment:
This will print the warning if the second bond is not found, and say its a bitcoin bond since ID = 0.